src/BaseDraw.php

```diff
@@ -785,6 +785,7 @@ public function getLegendSize(array $Format = [])
                     }
 
                     $vX = max($Width) + $XStep;
+                    unset($Width);
```

Comment:
Personally I prefer to be explicit here, because `$Width` below is an `int`, but I removed the line nevertheless.